sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java

```diff
@@ -1372,7 +1549,10 @@ public PCollectionView<Instant> expand(PBegin input) {
                 ParDo.of(rampupThrottlingFn).withSideInputs(startTimestampView));
       }
       intermediateOutput.apply(
-          "Write Mutation to Datastore", ParDo.of(new DatastoreWriterFn(projectId, localhost)));
+          "Write Mutation to Datastore",
+          ParDo.of(
+              new DatastoreWriterFn(
+                  projectId, localhost, new V1DatastoreFactory(), new WriteBatcherImpl())));
```

Comment:
this should likely not pass the new DatastoreFactory and WriteBatcher, as those already have defaults

Reply:
Done.